docs/orphaned_and_conflicting.md

```diff
@@ -0,0 +1,72 @@
+
+## What they are
+
+Yari relies on file structure/hierarchy being the same between `files/en-us/` from `mdn/content` and `files/<locale>/` from `mdn/translated-content`. However, from time to time, some differences occur and must be solved by a humain maintainer. Such differences are pushed into `files/<locale>/orphaned` and `files/<locale>/conflicting` directories.
+
+### Orphaned pages
+
+When a page does not exists anymore in English (or didn't exist at all, like it was possible to have in Kuma) but exists in a given locale, it is moved to `files/<locale>/orphaned`. The locale team should then decide if this content should be deleted/redirected or migrated to another page.
+
+In practice, deleting-redirecting per the `en-US` structure is enough (see below).
+
+### Conflicting pages
+
+When an English page is redirected/moved to a new location, but the target already exists in `<locale>`, the localized page is moved to `files/<locale>/conflicting`. Basically, a locale team member should decide which version take priority over the other.
+
+## How they are generated
+
+This "structure" synchronization between `mdn/content` and `mdn/translated-content` occurs daily with this GitHub Actions workflow https://github.com/nschonni/mdn-translated-content/blob/main/.github/workflows/sync-translated-content.yml (the logic can be found at https://github.com/mdn/yari/blob/main/build/sync-translated-content.ts)
+
+Before #8378 (16th Sep. 2022), this synchronization automatically added commits (see [an example](https://github.com/mdn/translated-content/commit/81b40438ed59fc3d5990eae0d1eb333ad7b5dd27), or [another](https://github.com/mdn/translated-content/commit/51268ca7b9bdf3ead1c1775dee61bcc90445f0dc)). Now, a PR is created with the changes for each locale (see #9029, #9026).
+
+## How to deal with them
+
+The goal of this process is to have 0 files under `files/<locale>/conflicting` / `files/<locale>/orphaned`. In the following sections, we assume that you setup both `mdn/content` and `mdn/translated-content` repositories on your local machine.
+
+### Deal with "first gen" files
+
+If your locale already contains a `conflicting` or `orphaned` directory, it has been populated by earlier sync commits. For each page, you will need to decide a course of action. Afterwards, rinse and repeat for the other pages.
+
+Depending on your team and on the directory you are processing, you may either go with a pull request dealing with only on page or with a PR for a small set of pages (e.g. processing the whole `conflicting/web/svg` directory at once if it contains 5-10 pages).
+
+#### Orphaned pages
+
+For an orphaned page, the generic approach consists of the following:
```

Comment:
For this part, I'm used to using command `git log /path/to/file/in/en-us/index.md` (or .md) to check which commit does the deletion belong to. So I can decide what should we do to deal with the orphaned pages.

And this is also helpful for conflicting pages. But for my bad english, I'm not sure how can we place this tip.

Reply:
Thanks a lot! I'll add this step (both for orphaned/conflicting). The more I think about it, the more I believe actually the same approach applies to both `conflicting` and `orphaned`

Reply:
And I'm thinking it might be possible/an improvement on the workflow to have a link to the commit(s) from the bot.

Reply:
Yes, agree. It's a good idea.

Reply:
I've changed accordingly and added the command (with `--` so that it works with deleted file)

Reply:
I'll file an issue to improve the sync PR message.